modules/nbde-secret-sharing-encryption.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// security/nbde-implementation-guide.adoc
+
+[id="nbde-secret-sharing-encryption_{context}"]
+= Secret sharing encryption
+
+Using Shamir’s secret sharing (sss), a cryptographic algorithm to securely divide up, distribute, and re-assemble keys, OpenShift can support more complicated mixtures of key protection.  When you configure OpenShift to use multiple Tang servers, OpenShift uses sss to allow the _Decrypt if at least one of the specified servers is available_ nature of the Network-Bound Disk Encryption (NBDE) disk encryption. You can create layers for additional security. For example, you can define a policy where OpenShift requires both the TPM and one of the given list of Tang servers to decrypt the disk.
```

Comment:
```suggestion
Using Shamir’s secret sharing (sss), a cryptographic algorithm to securely divide up, distribute, and re-assemble keys, {product-title} can support more complicated mixtures of key protection.  When you configure {product-title} to use multiple Tang servers, {product-title} uses sss to allow the _Decrypt if at least one of the specified servers is available_ nature of the Network-Bound Disk Encryption (NBDE) disk encryption. You can create layers for additional security. For example, you can define a policy where OpenShift requires both the TPM and one of the given list of Tang servers to decrypt the disk.
```

I strongly suggest reworking the first sentence. It's a bit hard to parse through.

Reply:
Changed -  Shamir’s secret sharing (sss) is a cryptographic algorithm to securely divide up, distribute, and re-assemble keys. Using this algorithm, OpenShift can support more complicated mixtures of key protection."